components/eamxx/src/share/grid/remap/horiz_interp_remapper_base.cpp

```diff
@@ -61,23 +61,24 @@ create_src_layout (const FieldLayout& tgt_layout) const
   using namespace ShortFieldTagsNames;
   const auto lt = get_layout_type(tgt_layout.tags());
   const bool midpoints = tgt_layout.has_tag(LEV);
-  const int vec_dim = tgt_layout.is_vector_layout() ? tgt_layout.dim(CMP) : -1;
+  const auto vec_tag = tgt_layout.is_vector_layout() ? tgt_layout.get_vector_tag() : INV;
+  const int vec_dim  = tgt_layout.is_vector_layout() ? tgt_layout.dim(vec_tag) : -1;
   auto src = FieldLayout::invalid();
   switch (lt) {
     case LayoutType::Scalar2D:
       src = m_src_grid->get_2d_scalar_layout();
       break;
     case LayoutType::Vector2D:
-      src = m_src_grid->get_2d_vector_layout(CMP,vec_dim);
+      src = m_src_grid->get_2d_vector_layout(vec_tag,vec_dim);
       break;
     case LayoutType::Scalar3D:
       src = m_src_grid->get_3d_scalar_layout(midpoints);
       break;
     case LayoutType::Vector3D:
-      src = m_src_grid->get_3d_vector_layout(midpoints,CMP,vec_dim);
+      src = m_src_grid->get_3d_vector_layout(midpoints,vec_tag,vec_dim);
       break;
     default:
-      EKAT_ERROR_MSG ("Layout not supported by CoarseningRemapper: " + e2str(lt) + "\n");
+      EKAT_ERROR_MSG ("Layout not supported by HorizInterpRemapperBase: " + e2str(lt) + "\n");
```

Comment:
Could we add `Target` to the beginning of this message and `Source` to the one below?  That way it is easier to see which of these two function threw the error.